t/meson.build

```diff
@@ -41,7 +42,6 @@ test('unit-tests', clar_unit_tests)
 
 unit_test_programs = [
   'unit-tests/t-example-decorate.c',
-  'unit-tests/t-hash.c',
```

Comment:
This whole commit should be merged into the preceding commit as it only contains fixups for it.